### DISCUSSION TITLE
Grandorgue prolongs any key

### DISCUSSION THREAD

**Discussion by CesPlayz** [GrandOrgue/grandorgue#1283] [Help]
sorry for bad english, when I leave grand orgue for about 30 seconds or so, when i press any key on the keyboard it will prolong that input unless I pressed that specific key again, and it will process normal again, it only happens when I leave it for about a second or minutes.

**Comment by mattrixed** [comment]
What operating system?
Version of GO?
What keyboard and how does GO "receive the keyboard input" ?  ie through a usb audio/midi device or a usb dongle to 5 din midi or straight usb connection from keyboard?

**Comment by CesPlayz** [comment]
Windows 11 grand orgue 3.8.0 i use printer cable connection from keyboard to laptop

**Comment by oleg68** [comment]
@CesPlayz 

Could you do Audio/Midi->Log MIDI events and post here the output when you peress and release a key and it continues playing
